pkg/operator/cassandra/controller/util/resource.go

```diff
@@ -303,6 +306,26 @@ func volumeMountsForRack(r cassandrav1alpha1.RackSpec, c *cassandrav1alpha1.Clus
 	return vm
 }
 
+func volumesForRack(r cassandrav1alpha1.RackSpec) []corev1.Volume {
+	volumes := []corev1.Volume{
+		{
+			Name: "shared",
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		},
+	}
+	if *r.JMXExporterConfigMapName != "" {
```

Comment:
This should be `if r.JMXExporterConfigMapName != nil && *r.JMXExporterConfigMapName != ""`